### PR TITLE
Users should be able to easily see earning potential

### PR DESCRIPTION
**Pull request by kcper**
### All Submissions

* [ ] Can you post a screenshot of your changes (if applicable)?

Tooltip:

https://user-images.githubusercontent.com/9114200/125789180-f7622544-f948-4e0c-a2cc-ac15bf445ba0.mp4

Resizing:

https://user-images.githubusercontent.com/9114200/125789509-ca7758b2-1314-41d2-bc95-0c3d6933d839.mp4

### New Feature Submissions

* [ ] Does this relate to an existing issue, or has it been talked about prior to being done (if a major change)?
https://github.com/CryptoBlades/cryptoblades/issues/235

### Notes

Please note, your code must pass all tests and lint checks before it can be merged.

### PR Description

Bonus shown in the tooltip is measured from the current level.
I'm not sure what Groshmak meant with '+ WeaponStats', so skipped for now.
Can we use that sword .png? I grabbed if from Groshmak's suggestion.

### CODE REVIEW

**Comment by seiyria** [comment]
I think it should probably wrap instead of shrink - `col-6 col-xs-12` - at smaller resolutions

**Comment by kcper** [comment]
Wait. That's just power bonus, I was supposed to show earnings bonus.

**Comment by kcper** [comment]
Ok now this thing shows actual reward gain based on hero power.
I removed `white-space: nowrap` so it now just wraps. I'm not sure where you want me to use the proposed style, because it breaks some alignments.

**Comment by seiyria** [comment]
@kcper that could be fine, can you send a screenshot of it big and small again with this new config?

File: frontend/src/components/smart/CharacterDisplay.vue [hunk 1]
```diff
@@ -130,7 +133,9 @@ export default {
       'ownCharacters',
       'timeUntilCharacterHasMaxStamina',
       'getIsInCombat',
-      'getIsCharacterViewExpanded'
+      'getIsCharacterViewExpanded',
+      'fightGasOffset',
+      'fightBaseline'
```
Comment (kcper) [inline frontend/src/components/smart/CharacterDisplay.vue:138]: fightGasOffset and fightBaseLine... I couldn't figure out how to use them in `methods: {` so had to pass from template html (as getters can be accessed only there) down to the methods.

**Comment by crypto-smoke** [comment]
Just a note that this will need to be updated when #283 is merged

**Comment by kcper** [comment]
image

**Comment by kcper** [comment]
> Just a note that this will need to be updated when #283 is merged

that's right!

**Comment by seiyria** [comment]
I'm confused, does it have to be updated? Aren't the same functions being called?

**Comment by kcper** [comment]
> I'm confused, does it have to be updated? Aren't the same functions being called?

the payout curve will change, that's why this will need to change
```
getAverageRewardAtLevel(level: number, fightGasOffset: number, fightRewardBaseling: number): number {
      return this.formattedSkill(fightGasOffset) + (this.formattedSkill(fightRewardBaseling) * (CharacterPower(level)/1000));
    },
```

**Comment by kcper** [comment]
image
image

**Review by seiyria — APPROVED** [review]

**Pull request merged**